h2/src/docsrc/help/help.csv

```diff
@@ -1763,6 +1763,7 @@ columnName dataType
 [ { AUTO_INCREMENT | IDENTITY } [ ( startInt [, incrementInt ] ) ] ]
 [ SELECTIVITY selectivity ] [ COMMENT expression ]
 [ PRIMARY KEY [ HASH ] | UNIQUE ] [ CHECK condition ]
+[ INVISIBLE ]
```

Comment:
We have to be as close to Oracle as possible with respect to syntax. Looks like `INVISIBLE` must go right after the column type.

https://docs.oracle.com/database/121/SQLRF/img_text/column_definition.htm